pyproject.toml

```diff
@@ -36,6 +36,9 @@ classifiers=[
   "Topic :: Utilities",
 ]
 
+[tool.setuptools.package-data]
+mplib = ["examples/*.md", "examples/*.png"]  # include all md files in mplib/examples
```

Comment:
Seemed like there're no *.md and *.png files under mplib/examples. Is this a typo that can be removed?